src/translations/en.json

```diff
@@ -4,7 +4,7 @@
     "addNewContencyFile": "Name of the new list",
     "parameters": "Parameters",
     "General": "General",
-    "theme": "Theme",
+    "theme": "Theme: ",
```

Comment:
There is a space before ':' in the french translation

Reply:
Yes, but in english, there is no space before ":"

Reply:
ok